project/newEvaluation.html

```diff
@@ -12,6 +12,7 @@
 
   <script src="assets/less.min.js" type="text/javascript"></script>
   <script src="src/views/common/common.js" type="text/javascript"></script>
+  <script src="src/data/new-evaluations.js" type="text/javascript"></script>
```

Comment:
This should be `new-evaluation.js`, presumably, right? In fact, I think I'd recommend calling it `create-evaluation.js` or maybe just plain `evaluation.js` instead (I've always had a bone to pick with temporal references included as part of property / method / file names - they're just confusing and easily become out of date).

Reply:
👍